espnet/asr/pytorch_backend/recog.py

```diff
@@ -18,6 +18,8 @@
 from espnet.utils.deterministic_utils import set_deterministic_pytorch
 from espnet.utils.io_utils import LoadInputsAndTargets
 
+from espnet2.bin.asr_inference import plot_attentions
```

Comment:
Is the reference to espnet2 not going to be an issue for the docker?

Reply:
It might be- I plan to move the plotting function elsewhere- maybe espnet/nets/asr_utils.py ?

Reply:
I'm not sure, @Fhrozen What do you think?

Reply:
Line 21 will be a problem for docker (egs). I would recommend to avoid calling from espnet2 -> espnet1.
in the case of moving to espnet/nets/asr_utils, then it should not be a problem.

Reply:
Thanks- will do